libs/cart/src/facades/cart/cart.facade.ts

```diff
@@ -50,6 +50,8 @@ export class DaffCartFacade<
 	orderResult$: Observable<V>;
 	orderResultId$: Observable<V['id']>;
 
+	private selectCartItemDiscountedTotal = getDaffCartSelectors<T, V>().selectCartItemDiscountedRowTotal;
```

Comment:
I would initialize this field in the constructor since we're already calling `getDaffCartSelectors` in there.

Reply:
I can't reference it outside of the constructor if I do that

Reply:
hm that doesn't sound right...the only change is where you initialize the value of the field, not the field declaration itself. It seems to work for me, doing `this.selectCartItemDiscountedTotal = selectCartItemDiscountedRowTotal` in the constructor.

Reply:
like that?

Reply:
yeah that looks good